src/diffTool/engineOperators.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { Operator } from "json-rules-engine";
+
+const customOperators: Operator[] = [];
```

Comment:
this file is "engineOperators" but only exports a "customOperators" as a default.

I remove the default and export customOperators here as a named export.  That way we support multiple types of operators.  OR change the filename/const name to match each other and maintain a single export. Otherwise, I see this can be confusing.

Reply:
I will rename it to customOperators.ts